### PR TITLE
Function toolbar changes

### PR DESCRIPTION
**Pull request by mavarius**
Closes #influxdata/applications-team-issues/issues/236
Closes #influxdata/applications-team-issues/issues/237
Closes #influxdata/applications-team-issues/issues/244
Closes #influxdata/applications-team-issues/issues/246

_Briefly describe your proposed changes:_
Change titles of Threesizer columns
Add search bar to filter functions
Fix issue where clicking the tooltip added the function to script
Change Text for documentation link
Add new functions provided from docs
Fix issue where caret and tooltip were misaligned

 - [x] Rebased/mergeable
 - [ ] Tests pass

### CODE REVIEW

File: ui/src/flux/components/flux_functions_toolbar/SearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  public componentWillMount() {
+    const waitPeriod = 100
+    this.handleSearch = _.debounce(this.handleSearch, waitPeriod)
+  }
+
+  public render() {
+    return (
+      <div className="flux-functions-toolbar--search">
+        <div className="search-widget">
+          <input
```
Comment (alexpaxton) [inline ui/src/flux/components/flux_functions_toolbar/SearchBar.tsx]: Use the ui kit input, it has loading and error states, also has icons

File: ui/src/flux/components/flux_functions_toolbar/ToolbarFunction.tsx [hunk 2]
```diff
@@ -47,11 +47,10 @@ class ToolbarFunction extends PureComponent<Props, State> {
         ref={r => (this.functionRef = r)}
         onMouseEnter={this.handleHover}
         onMouseLeave={this.handleStopHover}
-        onClick={this.handleClickFunction}
       >
         {this.tooltip}
-        <dd>
-          {func.name}() {this.helperText}
+        <dd onClick={this.handleClickFunction}>
```
Comment (alexpaxton) [inline ui/src/flux/components/flux_functions_toolbar/ToolbarFunction.tsx]: semantic af

File: ui/src/shared/components/TimeMachine/FluxQueryMaker.tsx [hunk 3]
```diff
@@ -126,7 +126,7 @@ class FluxQueryMaker extends PureComponent<Props, State> {
         ),
       },
       {
-        name: 'Functions',
+        name: 'Flux Functions',
```
Comment (alexpaxton) [inline ui/src/shared/components/TimeMachine/FluxQueryMaker.tsx]: Nice

File: ui/src/flux/components/flux_functions_toolbar/SearchBar.tsx (hunk 1 above)
Comment (mavarius) [reply ui/src/flux/components/flux_functions_toolbar/SearchBar.tsx]: replaced

**Comment by mavarius** [comment]
This PR replaced by #4725

**Pull request closed without merging**